src/user/library/index.ts

```diff
@@ -102,7 +116,7 @@ async function transformCreative(
 async function createNotification(accessToken: string, createInput: CreateNotificationCreativeInput) {
   const response = await axios.post(
     `${process.env.REACT_APP_SERVER_ADDRESS}`.replace("v1", "graphql"),
-    JSON.stringify({query: print(CreateNotificationCreativeDocument), variables: { input: createInput }}),
+    JSON.stringify({query: print(CreateNotificationCreativeDocument), variables: {input: createInput}}),
```

Comment:
Are you planning to replace this with a proper apollo mutation call?

Reply:
Since a campaign needs an already existing creativeId to be passed in, as of right now that is not possible. But if you are fine with me drafting a task in order to do the work to create everything in one go properly, I can eventually get rid of this!